chore(PHP): Relnotes for PHP agent release 11.1.0.14 (#18480)
newrelic/docs-website: src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-11-1-0-14.mdx

---
subject: PHP agent
releaseDate: '2024-08-26'
version: 11.1.0.14
downloadLink: 'https://download.newrelic.com/php_agent/archive/11.1.0.14'
features: []
bugs: ['Fixes unresolved strlcpy symbol error on Linux platforms using glibc >= 2.39']
security: []
---

## New Relic PHP agent v11.1.0.14

### Bug Fixes

> * Fixes an issue with newrelic.so not able to resolve the strlcpy symbol on Linux using glibc version >= 2.39

Reviewer: ```suggestion
* Fixes an issue where `newrelic.so` was not able to resolve the `strlcpy` symbol on Linux using glibc version 2.39 or higher.
```
